resources/card_templates/SNET/snet_station.html

```
@@ -369,7 +369,8 @@
         </div>
         <div class="list-scroll">
             {% for s in list_stations %}
-            <div class="list-row {% if loop.index is even %}list-row-even{% else %}list-row-odd{% endif %} {% if s.shindo >= 0 %}list-row-triggered{% else %}list-row-inactive{% endif %}">
+            {# 触发高亮与计数一致：震度0起点 = 計測震度 >= -0.5 #}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)